packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -10,29 +10,24 @@ import classNames from 'classnames';
 import AuthContext from '../../contexts/AuthContext';
 import { commentBoxClassNames } from '../comments/common';
 import { commentDateFormat } from '../../lib/dateFormat';
-import { Button } from '../buttons/Button';
 import styles from './CommentBox.module.css';
 import { ProfilePicture } from '../ProfilePicture';
-import { ClickableText } from '../buttons/ClickableText';
 import Markdown from '../Markdown';
 import { RecommendedMentionTooltip } from '../tooltips/RecommendedMentionTooltip';
 import {
   fixHeight,
   UPDOWN_ARROW_KEYS,
-  useUserMention,
+  UseUserMention,
```

Comment:
Can we maybe make the props separate as `UseUserMentionProps`

Reply:
done